src/applications/personalization/profile360/tests/actions/paymentInformation.unit.spec.js

```diff
@@ -0,0 +1,61 @@
+import { expect } from 'chai';
+import sinon from 'sinon';
+
+import * as paymentInformationActions from '../../actions/paymentInformation';
+
+let oldFetch;
+
+const setup = () => {
+  oldFetch = global.fetch;
+  global.fetch = sinon.stub();
+  global.fetch.returns(
+    Promise.resolve({
+      headers: { get: () => 'application/json' },
+      ok: true,
+      json: () =>
+        Promise.resolve({
+          data: [],
+        }),
+    }),
+  );
+};
+
+const teardown = () => {
+  global.fetch = oldFetch;
+};
```

Comment:
Out of scope for this ticket, but it's such a widely-used pattern to mock `fetch` this way that we should probably centralize or create a helper for it.